database/mml_loader/mml_loader.py

```diff
@@ -119,7 +119,13 @@ def parse_gml(self, output_dir: str, year: str, size: str) -> Dict:
 
         with self.Session() as session:
             admin_regions = session.query(AdministrativeRegion).all()
-            au_codes = sorted([admin_region.value for admin_region in admin_regions])
+            region_codes = sorted(
+                [admin_region.value for admin_region in admin_regions]
+            )
+            municipalities = session.query(Municipality).all()
+            municipality_codes = sorted(
+                [municipality.value for municipality in municipalities]
+            )
```

Comment:
Pieni nitpick viilaus, jolla ei tässä mittakaavassa niin merkitystä, mutta voisi olla aavistuksen tehokkaampi:
```suggestion
            from sqlalchemy import select
            municipality_codes = session.scalars(select(Municipality.value).order_by(Municipality.value)).all()
```
Sama tuon ylemmän region_codes kanssa.

Reply:
Totta joo, tietysti näin.